packages/odyssey-react-mui/src/ScreenReaderText.tsx

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { memo, ReactNode } from "react";
+import { Box, visuallyHidden } from "./";
+
+export type ScreenReaderTextProps = {
+  /**
+   * The visually-hidden text.
+   */
+  children: ReactNode;
+};
+
+const style = { ...visuallyHidden };
```

Comment:
Might be good to add a comment here explaining why we did this because of some weird MUI `sx` typing issue.